### ISSUE TITLE
Attack from south west peninsula to south east altis

### ISSUE THREAD

**Issue by StefArma** [A3Antistasi/antistasiofficial#253]
Vehicles won't manage to pathfind the waypoint to arrive, they stay at base.

**Comment by Sparker95** [comment]
What is this? Is it just an assumption or you saw it happen? Why can't they find a way if there are roads everywhere?

**Comment by StefArma** [comment]
saw it, they had waypoint to the AA outpost in pyrgos gulf, they were facing the sea and couln't move from spawn location (they spawned ok)

**Comment by Sparker95** [comment]
Vehicles often get stuck in arma because their driving is broken AF. Maybe a different time they won't get stuck, nobody knows. It's not necessary a path finding problem. Was there really a way from origin to destination?

**Comment by StefArma** [comment]
yes, go north, then west trough gravia and then south to pyrgos then to the AA outpost

**Comment by headstone1990** [comment]
Confirm this issue, thought to write about it on the forum, but saw it here. In my case is reproducible in 100% of AFRF attack, because now there are only 2 bases with which they occur. Gravia base and Terisa airport, my territory is Pirgos and the whole south-western part of the Altis. And the interesting part is that this happens with helicopters too. They just hang over the airfield and do nothing.
